wallet/src/libwallet/internal/updater.rs

```diff
@@ -60,6 +60,18 @@ where
 	Ok(outputs)
 }
 
+/// Retrieve all of the transaction entries
```

Comment:
I'm sure you're already aware of it, but this just might need to accept start/count type arguments at some point for pagination.